plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnectorConfiguration.java

```diff
@@ -212,9 +212,9 @@ protected static URI dockerDaemonUri(final boolean isLinux, @NotNull final Map<S
             try {
                 userURI = new URI(host);
             } catch (URISyntaxException e) {
-                LOG.error(String.format("Unable to parse %s property with value %s", DOCKER_HOST_PROPERTY, host), e);
+                LOG.error(String.format("Unable to parse %s value of environment variable %s", DOCKER_HOST_PROPERTY, host), e);
```

Comment:
I'm not sure that order of formatted string are then correct with the syntax change
It will display 
    Unable to parse DOCKER_HOST value of environment variable http://foo
while I think you expect
    Unable to parse http://foo value of environment variable DOCKER_HOST